extensions/amp-ad-exit/0.1/config.js

```diff
@@ -138,3 +148,29 @@ function assertTarget(name, target, config) {
     }
   }
 }
+
+/**
+ * Checks whether a vendor is valid (i.e. listed in vendors.js and has
+ * transport/iframe defined.
+ * @param {string} vendor The vendor name that should be listed in vendors.js
+ */
+function assertVendor(vendor) {
+  user().assert(ANALYTICS_CONFIG &&
```

Comment:
Why check `ANALYTICS_CONFIG`?

Reply:
ANALYTICS_CONFIG contains a map, the keys of which constitute a list of valid amp-analytics vendors.
We don't want to allow amp-ad-exit's config to indicate that it depends on a response from a vendor which doesn't actually exist. Nor one that exists, but doesn't use iframe transport and thus couldn't possibly send any response.